src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -481,6 +503,11 @@ void manualRegistrationToolBox::disableSaveButtons(bool param)
     d->b_exportTransformation->setDisabled(param);
 }
 
+void manualRegistrationToolBox::disableComputeButton(bool param)
+{
+    d->b_computeRegistration->setDisabled(param);
+}
```

Comment:
(since my opinion was requested...)
I agree that future coding should be avoided, but I also agree that code homogeneity is desirable unless the code we are extending contains bad coding practices, in which case it's best not do the same.
(The following is just a style remark, I'm not requesting the change)
What bothers me is the function signature. A function named ```disableFoo```  should disable Foo, not take a parameter deciding if it will be disabled or enabled. It's better to use setDisabled (or enabled), like QT did. Also ```param``` isn't a good choice for a function parameter name. It doesn't mean anything. ```disable``` would have been better.